deployment/terraform/anonymized_database_dump_sheduled_task/docker/dump.sh

```
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+export DATE=`date +%Y%m%d-%H%M%S`
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)